resources/keywords/provision_dbinstance.resource

```diff
@@ -8,12 +8,20 @@ Library     ../../utils/scripts/util.py
 *** Variables ***
 ${engine}     ${EMPTY}
 ${invalid}      False
+${pstgInstClass}    db.r5.xlarge
+${allocStorage}    20
 
 *** Keywords ***
 User Navigates To Create Database Instance Screen From Database Access Page
     User Navigates To Database Access Under Data Services
     Wait Until Page Contains Element    ${titleDatabaseAccess}
-    Select Create Database Instance From Configure Button DropDown
+    Select Create Database Instance From Configuration Button DropDown Provision
```

Comment:
@ikanias This is been discussed on earlier comment

Reply:
This is something that had to be done and we discussed it. This is a Dev bug which forces us to select a value in the drop down list in order for the RDS engine to be sent successfully. If we do not do that, then there will be an error saying that the engine parameter was not sent. This is temporary and will be changed when the bug will be fixed.

Reply:
Can you please review as I recall version 0.5.0 uses "Configure" option and "Configuration" used in earlier versions.